### ISSUE TITLE
Crashes on a certain moment of a video

### ISSUE THREAD

**Issue by firewater2** [mpv-player/mpv#4369]
It happens at 46:00 of https://thepiratebay.org/torrent/14633582/BBC_Louis_Theroux_A_Different_Brain_720p_HDTV_x264-MVGroup

I tried on MPC-HC and the same happens there, but I'm reporting this just in case, because I think it'd be better if MPV deals with it other than crashing (closes itself, or pauses).

**Comment by wm4** [comment]
Please cut a sample of this file - so that it still crashes, but small enough to upload somewhere publicly. You can probably use mkvmerge.

**Comment by firewater2** [comment]
Can I upload the whole thing to Mega? I always loose a lot of time when trying to do things like this (extract a subtitle from a video, merge audio, etc).

**Comment by wm4** [comment]
Probably better than a torrent, if it works with `megadl`.

**Comment by firewater2** [comment]
It's a drive error. Got a cycle redundancy check on the file while seeding it.

**Comment by wm4** [comment]
That doesn't mean it should crash on it, unless your system is unstable in general.

**Comment by firewater2** [comment]
What should it do instead? My system is not unstable, this is the first time it has happened.

I realized something was wrong with the file because it happened today with another video, and I checked uTorrent and the very same file had stopped seeding because of whatever a CRC is. Uploading it to Mega didn't work because I got a "IO failed error: progressevent". My drive seems to be in good conditions according to chkdsk, and is relatively new (5 months), so I don't know what's going on. But anyway it seems to definitely be on my end.
